packages/blocks/src/root-block/edgeless/edgeless-root-spec.ts

```diff
@@ -71,3 +77,27 @@ export const EdgelessRootBlockSpec: BlockSpec<EdgelessRootBlockWidgetName> = {
     },
   },
 };
+
+export const PreviewEdgelessRootBlockSpec: BlockSpec = {
+  schema: RootBlockSchema,
+  service: EdgelessRootService,
+  view: {
+    component: literal`affine-edgeless-root`,
+  },
+  setup(slots: BlockSpecSlots) {
+    slots.viewConnected.on(
+      ({
+        component,
+        service,
+      }: {
+        component: BlockElement;
+        service: BlockService;
+      }) => {
+        // Does not allow the edgeless to display toolbar.
+        (component as EdgelessRootBlockComponent).disableComponents = true;
+        // Does not allow the user to move and zoom.
+        (service as EdgelessRootService).locked = true;
```

Comment:
@pengx17 Does this affect your peek centre functionality?

Reply:
not sure - it seems fine :/

Reply:
ok